### PR TITLE
fix(): changed msg for no skills analytics

### LINKED ISSUE #2857: Wrong Message in My Analytics
**Actual Behaviour** 
"Your skill has not been used, make sure to improve your skill to attract more users." message is shown to users who haven't created any skills yet.

**Expected Behaviour**
 should show "You haven't created any skill yet. [Create skill Button]"

**Steps to reproduce it**
1. Create a new account 
2. Go to My Analytics Section in Dashboard.

**Screenshots of the issue**
 
Screenshot from 2019-08-15 19-42-06

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by kpulkit29**
Fixes #2857 

Changes: Changed the message when no skills are created (for analytics).

Demo Link: https://pr-3065-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
Screenshot 2019-11-17 at 10 46 12 PM

### CODE REVIEW

**Comment by kpulkit29** [comment]
Okay I would remove the button and display the 'no skill created text' only when the user skills count is 0.

**Comment by kpulkit29** [comment]
@akshatnitd  'You have not created any skill text will now appear only when there are no skills' other the current text would appear as it might be possible that usage is 0 but no. of skills are not

File: src/components/cms/Dashboard/MyAnalytics.js [hunk 1]
```diff
@@ -107,6 +109,17 @@ class MyAnalytics extends Component {
             </div>
           </Container>
         )}
+        {userSkills === 0 && !loading && (
```
Comment (akshatnitd) [inline src/components/cms/Dashboard/MyAnalytics.js]: This block is redundant. Please check the condition before return and store the message in a variable. And if `message !== ''`, then display the message.

**Comment by kpulkit29** [comment]
@akshatnitd  I had made the requested changes. Also I had squashed the commits but had to take the latest pull after that otherwise it was showing changes in 10 files. Kindly review .

**Pull request merged**